tests/testkit_extras/helpers.rs

```diff
@@ -206,43 +209,35 @@ pub fn listunspent_entry(raw: &RawBitcoinTx, addr: &btc::Address, confirmations:
     })
 }
 
-pub fn block_hash_on_height(sandbox: &Sandbox, height: Height) -> Hash {
-    let blockchain = sandbox.blockchain_ref();
-    let snapshot = blockchain.snapshot();
-    let schema = Schema::new(&snapshot);
-    schema.block_hashes_by_height().get(height.0).unwrap()
-}
-
 /// Anchor genesis block using funding tx
-pub fn anchor_first_block(sandbox: &AnchoringSandbox) {
-    let client = sandbox.client();
+pub fn anchor_first_block(testkit: &mut AnchoringTestKit) {
```

Comment:
Nitpick: seems like this and similar functions could be converted into methods in `AnchoringTestKit`. Could be performed later.